### PR TITLE
simplify LedgerDB

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph 
cc @dannyroberts

### CODE REVIEW

File: corehq/form_processor/backends/couch/ledger.py [hunk 1]
```diff
@@ -13,24 +13,15 @@ def get_ledgers_for_case(self, case_id):
...
-        from corehq.apps.commtrack.models import StockState
         try:
-            return StockState.objects.get(
-                case_id=unique_ledger_reference.case_id,
-                section_id=unique_ledger_reference.section_id,
-                product_id=unique_ledger_reference.entry_id,
+            return StockTransaction.latest(
+                case_id = unique_ledger_reference.case_id,
+                section_id = unique_ledger_reference.section_id,
+                product_id = unique_ledger_reference.entry_id,
```
Comment (dimagimon) [inline corehq/form_processor/backends/couch/ledger.py]: E251 unexpected spaces around keyword / parameter equals

File: corehq/form_processor/backends/couch/ledger.py [hunk 2]
```diff
@@ -13,24 +13,15 @@ def get_ledgers_for_case(self, case_id):
...
 
     def _get_ledger(self, unique_ledger_reference):
-        from corehq.apps.commtrack.models import StockState
         try:
-            return StockState.objects.get(
-                case_id=unique_ledger_reference.case_id,
-                section_id=unique_ledger_reference.section_id,
-                product_id=unique_ledger_reference.entry_id,
+            return StockTransaction.latest(
+                case_id = unique_ledger_reference.case_id,
```
Comment (dimagimon) [inline corehq/form_processor/backends/couch/ledger.py]: E251 unexpected spaces around keyword / parameter equals

File: corehq/form_processor/backends/couch/ledger.py [hunk 3]
```diff
@@ -13,24 +13,15 @@ def get_ledgers_for_case(self, case_id):
...
+                section_id = unique_ledger_reference.section_id,
```
Comment (dimagimon) [inline corehq/form_processor/backends/couch/ledger.py]: E251 unexpected spaces around keyword / parameter equals

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
bump cc @NoahCarnahan

**Pull request merged**